modules/imonomyBidAdapter.js

```diff
@@ -0,0 +1,126 @@
+import * as utils from 'src/utils';
+import { registerBidder } from 'src/adapters/bidderFactory';
+
+const BIDDER_CODE = 'imonomy';
+const ENDPOINT = '//b.imonomy.com/openrtb/hb/00000';
+const USYNCURL = '//b.imonomy.com/UserMatching/b/';
+
+export const spec = {
+  code: BIDDER_CODE,
+
+  /**
+  * Determines whether or not the given bid request is valid. Valid bid request must have placementId and hbid
+  *
+  * @param {BidRequest} bid The bid params to validate.
+  * @return boolean True if this is a valid bid, and false otherwise.
+  */
+  isBidRequestValid: bid => {
+    return !!(bid && bid.params && bid.params.placementId && bid.params.hbid);
+  },
+  /**
+  * Make a server request from the list of BidRequests.
+  *
+  * @param {validBidRequests[]} - an array of bids
```

Comment:
JSDoc @type declaration is using incorrect type, change to:
`* @param {BidRequest[]} validBidRequests an array of bids` (Also either remove or add `-` to all JSDoc parameter names, you can choose either, just be consistent in your adapter file)